app/templates/registration/confirmation.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+    <div>
+        <h1 style="font-size: 48px ; text-align: center">Success</h1>
+        <div id="logo" style="text-align: center">
+            <img height="20%" width="20%" style="display:block; margin-left:auto; margin-right:auto" src="assets/images/main_logo.png">
+        </div>
+        <h4 style="text-align: center">
+            Thank you for registering to become a SHARE source!
```

Comment:
Are we referring to them as providers here?

However, this whole page is kind of temporary. Is there a place we want to suggest the users to go next? Also wording? etc.